lib/delta_tables_generator.rb

```diff
@@ -0,0 +1,51 @@
+require_relative './delta_tables_generator/logger'
+
+require 'active_support/notifications'
+require 'active_support/log_subscriber'
+
+module DeltaTablesGenerator
+  extend self # rubocop:disable Style/ModuleFunction
```

Comment:
How do you feel about just prefixing the methods with self and not using this extend?

Reply:
Needed some adjustment for the instrumenting part, but yeah, it's actually a better idea, thanks!